### PR TITLE
Security policy page

### LINKED ISSUE #3250: Found a possible security concern
Hey there!

I belong to an open source security research community, and a member (@porcupineyhairs) has found an issue, but doesn’t know the best way to disclose it.

If not a hassle, might you kindly add a `SECURITY.md` file with an email, or another contact method? GitHub [recommends](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) this best practice to ensure security issues are responsibly disclosed, and it would serve as a simple instruction for security researchers in the future.

Thank you for your consideration, and I look forward to hearing from you!

(cc @huntr-helper)

### PR DESCRIPTION
**Pull request by northwestwitch**
Fixes partially #3250 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [ ] tests executed by

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report suspected security vulnerabilities to `chiara.rasi [at] scilifelab.se` or `daniel.nilsson [at] scilifelab.se`
```
Comment (dnil) [inline security.md:5]:
```suggestion
Please report suspected security vulnerabilities to `chiara.rasi [at] scilifelab.se` or `daniel.nilsson [at] scilifelab.se`

We warmly welcome all security feedback. We support the main branch and latest release with hot fixes as needed. Should you find a vulnerability for an older version, obviously please also be in touch, and we will see what we can do, especially if it is one that you know to be deployed in production somewhere.
```

Comment (dnil) [inline security.md:5]: 👍

**Review by dnil — APPROVED** [review]
Nice! The google guideline suggested also writing what versions we support.

File: security.md (hunk 1 above)
Comment (northwestwitch) [reply security.md:5]: Good suggestion!

**Pull request merged**